src/openrct2-ui/UiContext.Win32.cpp

```diff
@@ -62,28 +62,28 @@ namespace OpenRCT2::Ui
     public:
         Win32Context()
         {
-            _win32module = GetModuleHandleA(nullptr);
+            _win32module = GetModuleHandle(nullptr);
```

Comment:
Why are you changing these to the macro versions?

Reply:
Signaling that the charset doesn't matter for those, since they don't use strings either way.